service/frontend/handler.go

```diff
@@ -117,6 +124,8 @@ func NewWorkflowHandler(
 		hSerializerFactory: persistence.NewHistorySerializerFactory(),
 		domainCache:        cache.NewDomainCache(metadataMgr, sVice.GetLogger()),
 		rateLimiter:        common.NewTokenBucket(config.RPS, common.NewRealTimeSource()),
+		kafkaProducer:      kafkaProducer,
```

Comment:
Let's not use any Kafka naming at this layer.

Reply:
reduced to just the domainReplicator dependency